code/game/gamemodes/dynamic/dynamic_midround_rolling.dm

```diff
@@ -28,9 +29,10 @@
 
 	var/spawn_heavy = prob(get_heavy_midround_injection_chance())
 
-	last_midround_injection_attempt = world.time
-	next_midround_injection = null
-	forced_injection = FALSE
+	if(!forced)
+		last_midround_injection_attempt = world.time
+		next_midround_injection = null
+		forced_injection = FALSE
```

Comment:
if you're going to straight up bypass the forced_injection global, you should change the admin tools to use the parameter instead if possible

Reply:
Proc reverted to pre-PR state